ui/plugins/bookmarks/src/components/entry-renderer.tsx

```diff
@@ -122,9 +122,9 @@ const EntryCardRenderer = (props: IEntryCardRendererProps) => {
   }, [t, type]);
 
   const reportedTypeName = React.useMemo(() => {
-    if (accountAwaitingModeration) return `the author of this ${itemTypeName}`;
-    return `this ${itemTypeName}`;
-  }, [accountAwaitingModeration, itemTypeName]);
+    if (accountAwaitingModeration) return t(`the author of this ${itemTypeName}`);
+    return t(`this ${itemTypeName}`);
```

Comment:
I don't think that this can be extracted, apply the `t` only over the plain string without the `${itemTypeName}`

Reply:
And concatenate them afterwards